Tribler/community/allchannel2/structures.py

```diff
@@ -0,0 +1,261 @@
+import os
+
+from libtorrent import add_files, bdecode, bencode, create_torrent, file_storage, set_piece_hashes
+
+
+PIECE_SIZE = 16*1024*1024 # 16 MB: Holds 762600 magnetlinks without metadata
```

Comment:
Unclear to me what this does. Is this re-doing TFTP somewhat?

Reply:
This most likely refers to the torrent piece size, which seems to be fixed to 16MB for this implementation.